index.html

```diff
@@ -878,8 +881,17 @@ <h3>
           <li>Initialize |buttons| to be an empty [=list=].
           </li>
           <li>[=list/For each=] |buttonIndex:long| of [=the range=] from 0 to
-          |buttonsSize| − 1, [=list/append=] a [=new=] {{GamepadButton}} to
-          |buttons|.
+          |buttonsSize| − 1:
+            <ol>
+              <li>Let |button:GamepadButton| be a [=new=] {{GamepadButton}} with its
+              {{GamepadButton/type}} attribute initialized to |type|,
+              button.value initialized to 0,
+              button.pressed initialized to false, and
+              button.touched initialized to false.
```

Comment:
Let's use the same button.attr syntax for all four attributes and reorder them to match the order in the GamepadButton interface definition:

```
              <li>Let |button:GamepadButton| be a [=new=] {{GamepadButton}} with
              |button|.{{GamepadButton/pressed}} initialized to `false`,
              |button|.{{GamepadButton/touched}} initialized to `false`,
              |button|.{{GamepadButton/type}} initialized to |type|, and
              |button|.{{GamepadButton/value}} initialized to 0.
              </li>
```

Reply:
@nondebug  Thanks Matt. Just updated.